src/sync.ts

```diff
@@ -642,9 +634,6 @@ export class SyncApi {
             }
             this.opts.filter.setLazyLoadMembers(true);
         }
-        if (this.opts.lazyLoadMembers) {
-            this.syncOpts.crypto?.enableLazyLoading();
-        }
```

Comment:
Do we have documentation anywhere that confirms rust-crypto doesn't need something similar to support lazy loading being enabled?

Reply:
I don't remember the details right now, but no this is not required.